doc/source/user_guide_scripting/helper_scripts.rst

```diff
@@ -0,0 +1,342 @@
+- :ansyshelp:`Get the movement and force of a joint probe <act_script_demo_cylindrical_joint.html>`
+- :ansyshelp:`Add an energy probe (RBD) <act_script_demo_rbd_contact.html>`
+
+------------------------
+
+- :ansyshelp:`Get modal natural frequencies <act_script_demo_random_vib.html>`
+- :ansyshelp:`Add the deformation frequency response <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add the voltage frequency response <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add the charge reaction frequency response <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add the impedance frequency response <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Get the maximum value from a frequency response <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add PSD results - Directional Deformation, Velocity, Acceleration <act_script_demo_random_vib.html>`
+- :ansyshelp:`Add response PSD <act_script_demo_random_vib.html>`
+- :ansyshelp:`Add response PSD tool <act_script_demo_random_vib.html>`
+
+------------------------
+
+- :ansyshelp:`Add a radiation probe <act_script_demo_thermal_electric.html>`
+- :ansyshelp:`Add a temperature result <act_script_demo_steady_state_therm.html>`
+- :ansyshelp:`Add a total heat flux result <act_script_demo_steady_state_therm.html>`
+- :ansyshelp:`Add a convection BC reaction probe <act_script_demo_steady_state_therm.html>`
+
+------------------------
+
+- :ansyshelp:`Add an acoustic pressure result <act_script_demo_harmonic_acoustic.html>`
+- :ansyshelp:`Add an acoustic SPL result <act_script_demo_harmonic_acoustic.html>`
+- :ansyshelp:`Add an acoustic far field SPL result <act_script_demo_harmonic_acoustic.html>`
+- :ansyshelp:`Add an acoustic far field weighted SPL result <act_script_demo_harmonic_acoustic.html>`
+
+------------------------
+
+- :ansyshelp:`Add an electric directional e-field intensity <act_script_demo_electric.html>`
+- :ansyshelp:`Add an electric directional current density <act_script_demo_electric.html>`
+- :ansyshelp:`Add an electric directional EMAG reaction force <act_script_demo_electric.html>`
+- :ansyshelp:`Add an electric voltage result <act_script_demo_electric.html>`
+- :ansyshelp:`Add electric joule heat <act_script_demo_electric.html>`
+
+------------------------
+
+- :ansyshelp:`Clear generated data <act_script_demo_post1.html>`
+- :ansyshelp:`Evaluate all results <act_script_demo_post2.html>`
+- :ansyshelp:`Duplicate a harmonic result object <act_script_examples_duplicate_result_object.html>`
+- :ansyshelp:`Scan results, suppress results with invalid display times, and evaluate the results <act_script_examples_scan_results.html>`
+- :ansyshelp:`Rename results based on definition <act_script_demo_post2.html>`
+- :ansyshelp:`Modify display options for inserted results <act_script_demo_post2.html>`
+- :ansyshelp:`Modify display options for user defined results <act_script_demo_post2.html>`
```

Comment:
same here with saying it should be user-defined results (but go ahead and leave if that is what the help title looks like).